src/lib.rs

```diff
@@ -1638,34 +1654,53 @@ impl Bindings {
             cmd.args(&["--config-path", path]);
         }
 
-        if let Ok(output) = cmd.arg(file).output() {
-            if !output.status.success() {
-                let stderr = String::from_utf8_lossy(&output.stderr);
-                match output.status.code() {
-                    Some(2) => Err(io::Error::new(
-                        io::ErrorKind::Other,
-                        format!("Rustfmt parsing errors:\n{}", stderr),
-                    )),
-                    Some(3) => {
-                        warn!(
-                            "Rustfmt could not format some lines:\n{}",
-                            stderr
-                        );
-                        Ok(())
+        if let Ok(mut child) = cmd.spawn() {
+
+            let mut child_stdin = child.stdin.take().unwrap();
+            let mut child_stdout = child.stdout.take().unwrap();
```

Comment:
@fitzgen Is there a better way than to unwrap here?

Reply:
No, unwrap is fine because we know it is `Some` becuse we asked it to be `Stdio::piped()`